controller.go

```diff
@@ -130,6 +130,9 @@ func NewController(
 					case "packet":
 						provisioner, _ = provision.NewPacketProvisioner(controller.infraConfig.GetAccessKey())
 						break
+					case "scaleway":
+						provisioner, _ = provision.NewScalewayProvisioner(controller.infraConfig.GetAccessKey(), controller.infraConfig.SecretKey, controller.infraConfig.OrganizationID)
```

Comment:
Scaleway call this organisationID with the British spelling, we should follow the same.

Reply:
Where did you see that they are using the British spelling? 
On the sdk: `func (c *Client) GetDefaultOrganizationID() (organizationID string, exists bool)`
On the console: 
<img width="197" alt="image" src="https://user-images.githubusercontent.com/12498987/68025268-abe50c00-fcac-11e9-8942-b136ac4d6e60.png">